config.yml

```diff
@@ -319,25 +319,25 @@ steps:
     link: '{{ repoUrl }}/issues/10'
     actions:
       - type: gate
-        left: '848cd8c, 56d6fbb'
-        operator: test
+        left: '/(Add .env file)|(Remove .env file)/g'
+        operator: '!test'
```

Comment:
Hey @hectorsector, the previous command removes the commit contents relating to the .env file, it doesn't remove the entire commits from history. I had this validation against the original commit IDs because those would change with the command but i'm open to a better way for validation.

However, with this validation and because the commits along with their commit messages remain, I don't get the last bot response from this step that sends me to the congratulations issue.

Reply:
OK I understand now, that's not clear from the responses so I'll make those adjustments and then merge.